packages/playback-core/src/types.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint @typescript-eslint/triple-slash-reference: "off" */
-/// <reference path="../../../node_modules/mux-embed/dist/types/mux-embed.d.ts" />
+/// <reference path="../../../node_modules/mux-embed/dist/types/mux-embed.d.ts" preserve="true" />
```

Comment:
NOTE: Now need to add `preserve="true"` to ensure definitions for mux-embed are preserved in dist. See: https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#simplified-reference-directive-declaration-emit